Dockerfile.aarch64

```diff
@@ -10,11 +10,9 @@ LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DA
 LABEL maintainer="aptalca"
 
 RUN \
-  echo "**** install build packages ****" && \
+echo "**** install build packages ****" && \
```

Comment:
Indent got messed up

Reply:
vscode's copy/paste indent behavior changed and I really don't like it. It's always wrong and I have to manually fix it every time

fixed